connectors/agentic-ai/element-templates/agenticai-mcp-client-outbound-connector.json

```diff
@@ -0,0 +1,203 @@
+{
+  "$schema" : "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name" : "MCP Client (alpha)",
+  "id" : "io.camunda.connectors.agenticai.mcp.client.v0",
```

Comment:
Tagging this as alpha/v0 for the moment - will update to v1 as soon as we have auth support and docs.